packages/wdio-allure-reporter/tests/runtime.test.js

```diff
@@ -10,6 +10,12 @@ describe('reporter reporter api', () => {
         jest.resetAllMocks()
     })
 
+    it.only('should pass correct data from addLabel', () => {
```

Comment:
```suggestion
    it('should pass correct data from addLabel', () => {
```
Removing this will fix the coverage report